Router.svelte

```diff
@@ -76,7 +76,7 @@ export function push(location) {
 
     // Execute this code when the current call stack is complete
     setTimeout(() => {
-        window.location.hash = location
+        window.location.hash = location.charAt(0) == '#' ? location : '#' + location
```

Comment:
Wait this won't work either. In line 73 we are checking to make sure that location.charAt(0) is '/'.

Need to update line 73 to:

````js
if (!location || location.length < 1 || (location.charAt(0) != '/' && location.indexOf('#/') !== 0)) {
````